### ISSUE TITLE
✅ 🔃 🐞 Bug Report: ACF fields not working (Merged with #20235)

### ISSUE THREAD

**Issue by digitalmurph** [elementor/elementor#20015]
### Prerequisites

- [X] I have searched for similar issues in both open and closed tickets and cannot find a duplicate.
- [X] The issue still exists against the latest stable version of Elementor.

### Description

In the latest versio of Elementor Pro, Advanced Custom Fields stop displaying completely on the font end, and they are not visible as an option in the Elementor Builder.

Rolling back to Elementor Pro 3.6.0 fixes the issue.

### Steps to reproduce

1) Up to date versions of Elemmentor Pro and ACF - custom fields disappear
2) Rollback to Elementor Pro 3.6.0 and the problem is gone - ACF fields reappear.

### Isolating the problem

- [X] This bug happens with only Elementor plugin active (and Elementor Pro).
- [X] This bug happens with a Blank WordPress theme active ([Hello theme](https://wordpress.org/themes/hello-elementor/)).
- [X] I can reproduce this bug consistently using the steps above.

### System Info

<details>
<summary>System Info</summary>

````txt


					
== Server Environment ==
	Operating System: Linux
	Software: Flywheel/5.1.0
	MySQL version: (Google) v5.7.38
	PHP Version: 7.4.32
	PHP Memory Limit: 128M
	PHP Max Input Vars: 10000
	PHP Max Post Size: 300M
	GD Installed: Yes
	ZIP Installed: Yes
	Write Permissions: There are some writing permissions issues with the following directories/files:
		 - WordPress root directory
	Elementor Library: Connected

== WordPress Environment ==
	Version: 6.0.2
	Site URL: https://pixieschildcare.co.uk
	Home URL: https://pixieschildcare.co.uk
	WP Multisite: No
	Max Upload Size: 300 MB
	Memory limit: 128M
	Max Memory limit: 128M
	Permalink Structure: /%postname%/
	Language: en-US
	Timezone: 0
	Admin Email: pixieschildcarelimited@hotmail.co.uk
	Debug Mode: Inactive

== Theme ==
	Name: Pixies CHildcare
	Version: 3.0.1.1611842743
	Author: Brainstorm Force
	Child Theme: Yes
	Parent Theme Name: Astra
	Parent Theme Version: 3.9.2
	Parent Theme Author: Brainstorm Force

== User ==
	Role: administrator
	WP Profile lang: en_US
	User Agent: Mozilla/5.0 (Macintosh; Intel Mac OS X 10.15; rv:105.0) Gecko/20100101 Firefox/105.0

== Active Plugins ==
	Advanced Custom Fields
		Version: 6.0.2
		Author: WP Engine

	Better Search Replace
		Version: 1.4.1
		Author: Delicious Brains

	Clear Cache For Me
		Version: 1.8
		Author: Webhead LLC

	Custom Post Type UI
		Version: 1.13.1
		Author: WebDevStudios

	Disable Gutenberg
		Version: 2.8
		Author: Jeff Starr

	Duplicate Page
		Version: 4.4.9
		Author: mndpsingh287

	DynamicConditions
		Version: 1.6.0
		Author: RTO GmbH

	Ele Custom Skin
		Version: 3.1.7
		Author: Dudaster.com

	Elementor
		Version: 3.9.0-dev2
		Author: Elementor.com

	Elementor Beta (Developer Edition)
		Version: 1.1.1
		Author: Elementor.com

	Elementor Pro
		Version: 3.7.4
		Author: Elementor.com

	Make Column Clickable Elementor
		Version: 1.4.0
		Author: Fernando Acosta

	Make Elementor Default Editor
		Version: 0.1.4
		Author: Fletcher Digital

	Post Types Order
		Version: 1.9.9.1
		Author: Nsp Code

	Redirection
		Version: 5.3.4
		Author: John Godley

	WP Activity Log
		Version: 4.4.2.1
		Author: WP White Security

	WP Clean Admin Menu
		Version: 2.0
		Author: P. Roy

	WP Table Builder
		Version: 1.4.3
		Author: WP Table Builder


== Elementor Experiments ==
	Optimized DOM Output: Active by default
	Improved Asset Loading: Active by default
	Improved CSS Loading: Active by default
	Inline Font Icons: Inactive by default
	Accessibility Improvements: Active by default
	Additional Custom Breakpoints: Active by default
	Import Export Website Kit: Active
	Hide native WordPress widgets from search results: Active by default
	admin_menu_rearrangement: Inactive by default
	Flexbox Container: Active by default
	Default to New Theme Builder: Active by default
	Landing Pages: Active by default
	Color Sampler: Active by default
	Favorite Widgets: Active by default
	Admin Top Bar: Active by default
	Page Transitions: Active by default
	Notes: Active by default
	Form Submissions: Active by default
	Scroll Snap: Active by default


== Log ==
	
Log: showing 20 of 402022-09-22 12:33:06 [info] Elementor/Upgrades - _v_3_8_0_fix_php8_image_custom_size Start  
2022-09-22 12:33:06 [info] Elementor/Upgrades - _v_3_8_0_fix_php8_image_custom_size Finished 
2022-09-22 12:33:06 [info] Elementor data updater process has been queued. [array (
  'plugin' => 'Elementor',
  'from' => '3.8.0-dev1',
  'to' => '3.9.0-dev1',
)]
2022-09-22 12:33:06 [info] Elementor data updater process has been completed. [array (
  'plugin' => 'Elementor',
  'from' => '3.8.0-dev1',
  'to' => '3.9.0-dev1',
)]
2022-09-26 15:01:20 [info] elementor-pro::elementor_pro_updater Started 
2022-09-26 15:01:20 [info] Elementor Pro/Upgrades - _on_each_version Start  
2022-09-26 15:01:20 [info] Elementor Pro/Upgrades - _on_each_version Finished 
2022-09-26 15:01:20 [info] Elementor data updater process has been completed. [array (
  'plugin' => 'Elementor Pro',
  'from' => '3.8.0-dev31',
  'to' => '3.9.0-dev1',
)]
2022-10-10 08:49:29 [info] Elementor data updater process has been queued. [array (
  'plugin' => 'Elementor',
  'from' => '3.9.0-dev1',
  'to' => '3.9.0-dev2',
)]
2022-10-10 08:49:30 [info] Elementor data updater process has been queued. [array (
  'plugin' => 'Elementor Pro',
  'from' => '3.9.0-dev1',
  'to' => '3.9.0-dev2',
)]
2022-10-10 08:49:30 [info] elementor::elementor_updater Started 
2022-10-10 08:49:31 [info] Elementor/Upgrades - _on_each_version Start  
2022-10-10 08:49:31 [info] Elementor/Upgrades - _on_each_version Finished 
2022-10-10 08:49:31 [info] Elementor data updater process has been completed. [array (
  'plugin' => 'Elementor',
  'from' => '3.9.0-dev1',
  'to' => '3.9.0-dev2',
)]
2022-10-10 08:49:31 [info] elementor-pro::elementor_pro_updater Started 
2022-10-10 08:49:31 [info] Elementor Pro/Upgrades - _on_each_version Start  
2022-10-10 08:49:31 [info] Elementor Pro/Upgrades - _on_each_version Finished 
2022-10-10 08:49:31 [info] Elementor data updater process has been completed. [array (
  'plugin' => 'Elementor Pro',
  'from' => '3.9.0-dev1',
  'to' => '3.9.0-dev2',
)]
2022-10-10 08:49:33 [info] Elementor data updater process has been queued. [array (
  'plugin' => 'Elementor',
  'from' => '3.9.0-dev1',
  'to' => '3.9.0-dev2',
)]
2022-10-10 08:49:34 [info] Elementor data updater process has been queued. [array (
  'plugin' => 'Elementor Pro',
  'from' => '3.9.0-dev1',
  'to' => '3.9.0-dev2',
)]

PHP: showing 2 of 2PHP: 2022-10-10 08:52:49 [notice X 29][/www/wp-content/plugins/elementor-pro/modules/forms/classes/form-record.php::179] Undefined index: referrer [array (
  'trace' => '
#0: Elementor\Core\Logger\Manager -> shutdown()
',
)]
PHP: 2022-10-10 12:34:01 [notice X 3][/www/wp-content/plugins/elementor/core/common/modules/ajax/module.php::171] Undefined index: data [array (
  'trace' => '
#0: Elementor\Core\Logger\Manager -> shutdown()
',
)]



== Elementor - Compatibility Tag ==
	
	Elementor Pro: Compatibility not specified
	Make Column Clickable Elementor: Compatibility not specified
	Make Elementor Default Editor: Compatibility not specified

== Elementor Pro - Compatibility Tag ==
	

				
````

</details>

**Comment by caspervoogt** [comment]
I am also experiencing this, though I *can* select ACF fields with the builder, they do not show up.

**Comment by bartderuyck** [comment]
This happens to me, too.

The custom fields can be set inside the editor but disappear as soon as the block or editor state changes. For example, changing the styling of the field, editing the site settings, or exiting the builder.

Reverting to 3.7.6 does indeed fix the issue.

**Comment by nicholaszein** [comment]
Hey @digitalmurph, 

Thank you for the feedback. 🙏 

⚠️ While reviewing your report, we noticed it was missing some key pieces of information:

* **Steps to replicate**: Provide us with enough information to be able to reproduce your issue consistently
* **Isolating the problem**: In your System Info we are seeing that you are using 3rd party add-ons that modify Elementor's behavior and are incompatible with the version of Elementor you're using. You're also using the Elementor Developers Edition, which is a plugin that shouldn't be used on production websites. It's only for developers to test their addons with the latest, _unstable_ Elementor features. You have also not tested with a blank theme either, as required in our guidelines. Please make sure to isolate all the possible conflicts before submitting your report.

Without them, we simply can't help. 😕 

👉 Please follow the guidelines for opening a [new issue report](https://github.com/elementor/elementor/issues/new?assignees=&labels=status%2Fawaiting_triage&template=bug.yml), so we can help you better.

👍 Please feel free to open a new issue in accordance with these guidelines.

@caspervoogt and @bartderuyck, please submit your issues in the same fashion so we can verify if it's an Elementor bug.

Best regards,

Nick

**Comment by nicholaszein** [comment]
Hello @digitalmurph, @caspervoogt, and @bartderuyck!

Thank you for taking the time to report this to us. 🙏 



### ⛔ While reviewing your report we concluded this is not an issue caused by Elementor. Here's what we found out:
- I could consistently update Elementor to the latest version and see all fields appearing as expected.

### You can verify my System Info
<details>
<summary>System Info</summary>
		
== Server Environment ==
	Operating System: Linux
	Software: nginx/1.21.4
	MySQL version: mariadb.org binary distribution v10.5.17
	PHP Version: 7.4.32
	PHP Memory Limit: 256M
	PHP Max Input Vars: 1000
	PHP Max Post Size: 256M
	GD Installed: Yes
	ZIP Installed: Yes
	Write Permissions: All right
	Elementor Library: Connected

== WordPress Environment ==
	Version: 6.0.3
	Site URL: localhost
	Home URL: localhost
	WP Multisite: No
	Max Upload Size: 256 MB
	Memory limit: 40M
	Max Memory limit: 256M
	Permalink Structure: /%postname%/
	Language: en-US
	Timezone: America/Sao_Paulo
	Admin Email: redacted
	Debug Mode: Active

== Theme ==
	Name: Hello Elementor
	Version: 2.6.1
	Author: Elementor Team
	Child Theme: No

== User ==
	Role: administrator
	WP Profile lang: en_US
	User Agent: Mozilla/5.0 (Windows NT 10.0; Win64; x64) AppleWebKit/537.36 (KHTML, like Gecko) Chrome/106.0.0.0 Safari/537.36

== Active Plugins ==
	Advanced Custom Fields PRO
		Version: 6.0.3
		Author: WP Engine

	Elementor
		Version: 3.8.0
		Author: Elementor.com

	Elementor Pro
		Version: 3.8.0
		Author: Elementor.com


== Elements Usage ==
	
	wp-post : 0
		button : 1
		container : 1
		heading : 8
		image : 1
		image-gallery : 1


== Elementor Experiments ==
	Optimized DOM Output: Active by default
	Improved Asset Loading: Active by default
	Improved CSS Loading: Inactive
	Inline Font Icons: Inactive by default
	Accessibility Improvements: Active by default
	Additional Custom Breakpoints: Inactive
	Import Export Website Kit: Active by default
	Hide native WordPress widgets from search results: Active by default
	admin_menu_rearrangement: Inactive by default
	Flexbox Container: Active
	Default to New Theme Builder: Inactive
	Hello Theme Header & Footer: Inactive
	Landing Pages: Inactive
	Color Sampler: Active by default
	Favorite Widgets: Active by default
	Admin Top Bar: Active by default
	Page Transitions: Active by default
	Notes: Active by default
	Loop: Inactive by default
	Form Submissions: Inactive
	Scroll Snap: Active by default


== Log ==
	
Log: showing 13 of 132022-11-01 07:57:03 [info] Elementor data updater process has been queued. [array (
  'plugin' => 'Elementor',
  'from' => '3.7.7',
  'to' => '3.8.0',
)]
2022-11-01 07:57:03 [info] elementor-pro::elementor_pro_updater Started 
2022-11-01 07:57:03 [info] Elementor Pro/Upgrades - _on_each_version Start  
2022-11-01 07:57:03 [info] Elementor Pro/Upgrades - _on_each_version Finished 
2022-11-01 07:57:03 [info] Elementor data updater process has been completed. [array (
  'plugin' => 'Elementor Pro',
  'from' => '3.7.7',
  'to' => '3.8.0',
)]
2022-11-01 07:57:03 [info] Elementor data updater process has been queued. [array (
  'plugin' => 'Elementor Pro',
  'from' => '3.7.7',
  'to' => '3.8.0',
)]
2022-11-01 07:57:03 [info] elementor::elementor_updater Started 
2022-11-01 07:57:03 [info] Elementor/Upgrades - _on_each_version Start  
2022-11-01 07:57:03 [info] Elementor/Upgrades - _on_each_version Finished 
2022-11-01 07:57:03 [info] Elementor/Upgrades - _v_3_8_0_fix_php8_image_custom_size Start  
2022-11-01 07:57:03 [info] Elementor/Upgrades - _v_3_8_0_fix_php8_image_custom_size Finished 
2022-11-01 07:57:03 [info] Elementor data updater process has been completed. [array (
  'plugin' => 'Elementor',
  'from' => '3.7.7',
  'to' => '3.8.0',
)]
2022-11-01 07:57:04 [info] Elementor data updater process has been queued. [array (
  'plugin' => 'Elementor Pro',
  'from' => '3.7.7',
  'to' => '3.8.0',
)]

== Elementor - Compatibility Tag ==
	
	Elementor Pro: Compatible

== Elementor Pro - Compatibility Tag ==
</details>

__________________________________

### ✅ What you can do to solve your issue:

⚠️  **Backup** your website before proceeding, and if possible do it in a staging environment!

1. Disable/Deactivate all your plugins except **Elementor** (and **Elementor Pro**).
    - If this solves the problem, gradually re-activate plugin-by-plugin (and check if the problem returns) to identify the problematic or conflicting plugin. In this case, contact the 3rd-party plugin support as we don't provide support for 3rd party plugins.
    - If it doesn't solve your problem, try the next step:
2. Change your WordPress theme to the [Hello Elementor](https://wordpress.org/themes/hello-elementor/) theme and check if the problem is solved.
3. Clear all cache types, including but not limited to **Server cache**, **WordPress (optimization plugin) cache**, **CDN cache**, and **Browser cache**. Also, try visiting the front page of your website in **incognito/private** mode.
4. Make sure the server resources allocated to your site and WordPress installation meet our [Requirements](https://elementor.com/help/requirements/). To check this information go to your _WordPress Dashboard > Elementor > System Info_, and check both your **Server Environment** and **WordPress Environment** tables. You can also scroll down to the end of the **System Info** page and click on the **Download System Info** button. Then submit another comment here and drop the `.txt` file inside the text box to upload your system information and share it with us.

For now, that's all we can help with. If you think this is still an issue caused by Elementor, please feel free to open a [new issue report](https://github.com/elementor/elementor/issues/new?assignees=&labels=status%2Fawaiting_triage&template=bug.yml) and provide us with the most information you can possibly get by following our guidelines:

### ⚠️ Please Note that GitHub issues are reserved for 🐞**Bug Reports**, and should not be used for support.

* To **avoid duplicates**, please search this repository for similar issues before submitting your own.
* If the issue can't be recreated on other sites, it's not an Elementor bug. In that case, **use the support links below before submitting your issue here**.
* Make sure you are using the most **updated versions** of Elementor, WordPress & your theme.
* Deactivate all of your plugins. If this solves the problem, gradually activate your plugins one by one, until you spot the problematic plugin, and then **contact the 3rd party plugin developer**.
* Make sure to **provide a detailed description** of your issue.
* Make sure to **provide detailed steps to replicate** your issue.
* Make sure the issue occurs with **only Elementor** (and Elementor Pro) plugin(s) active.
* Make sure the issue happens with a **blank theme** active ([Hello Elementor](https://wordpress.org/themes/hello-elementor/)).
* Make sure to **provide a copy of your _Elementor > System Info_**. Download the `.txt` file and drag it inside the "System Info" field when submitting a new bug report.



❔ If you have questions or need general support, please use one of the following channels:

- [Elementor Global Community](https://facebook.com/groups/Elementors/) on Facebook
- [Elementor Help Center](https://docs.elementor.com)
- [Support Forum](https://wordpress.org/support/plugin/elementor) on WordPress.org 
- [Elementor Developers Documentation](https://developers.elementor.com/)

💎 If you have purchased [Elementor Pro](https://elementor.com/pro), you are entitled to personal support, please see your purchase email or visit your [Elementor Account page](https://my.elementor.com/) for details.

Kind regards

**Comment by rexrainey** [comment]
I'm having the same issue. When using ACF fields within a "Template" widget, the info will not show on the live page.

**Comment by reuhno** [comment]
Same issue here, and I know an other person who is having the problem. ACF fields suddenly stopped working in loops on front, this needs to be fixed ASAP

Meanwhile, rollback to 3.7.7...

**Comment by caspervoogt** [comment]
@nicholaszein I had run into this with a blank theme, plugins deactivated etc, with Elementor fully updated. No time to go back to that project and test further and besides, I recently updated that one to the latest Elementor and it behaved fine. Plus Elementor's releasing loop templates of its own, which may obviate the need for 3rd party plugins for that kind of thing.

**Comment by hookandloopAus** [comment]
Ditto. Elementor can see the ACF field's fallback setting, but the not the custom content of the field.

**Comment by nicholaszein** [comment]
Hello everybody!

We have great news! 🙌 

### 📢 We're happy to announce that the issue you raised was resolved in Elementor Pro **v3.8.1**! 🥳 

✔️ Feel free to check it out and update your plugin to the new version!

Cheers 🥂
